ThePorch/src/tabs/connect/ConnectScreenFeatures.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { View, Linking } from 'react-native';
+import { ActionListFeature as CoreActionListFeature } from '@apollosproject/ui-connected';
+import {
+  H2,
+  Touchable,
+  styled,
+  Icon,
+  PaddedView,
+} from '@apollosproject/ui-kit';
+import { withNavigation } from 'react-navigation';
+import SafeAreaView from 'react-native-safe-area-view';
+
+const HorizontalView = styled(({ theme }) => ({
+  flexDirection: 'row',
+  justifyContent: 'space-around',
+  alignItems: 'center',
+  marginBottom: theme.sizing.baseUnit,
+}))(View);
+
+const SocialIcon = styled(({ theme }) => ({
+  width: 50,
+  height: 50,
+  borderRadius: 50,
+  backgroundColor: theme.colors.darkPrimary,
+  justifyContent: 'center',
+  alignItems: 'center',
+}))(View);
+
+const StyledH2 = styled(({ theme }) => ({
+  fontSize: theme.sizing.baseUnit * 1.75,
+}))(H2);
+
+const SocialIconsFeature = ({ title, socialIcons }) => {
+  // TODO: Once `socialIcons` is working use that instead of `socialIconsTemp`. Doing this for testing purposes.
```

Comment:
??

Reply:
When I was fixing the styles for the social icons the icon was hard coded to the `play` icon and it didn't seem like the `socialIcons` from the api was wired up yet. I'll double check with Vincent in the morning to see if that is wired up now. Then I can kill my hard coded array below that comment.

Reply:
@conrad-vanl I pushed up fix to use the static `socialIcons` array from the api Vincent setup. TODO comment is gone. https://github.com/watermarkchurch/theporch-apollos/pull/30/commits/dba800238d7a2c02f14d069890512100723b1719